chromium_src/components/history_clusters/core/features.cc

```diff
@@ -17,8 +17,11 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kOmniboxAction, base::FEATURE_DISABLED_BY_DEFAULT},
     {kPersistedClusters, base::FEATURE_DISABLED_BY_DEFAULT},
     {kPersistContextAnnotationsInHistoryDb, base::FEATURE_DISABLED_BY_DEFAULT},
-    {kSidePanelJourneys, base::FEATURE_DISABLED_BY_DEFAULT},
 }});
 
 }  // namespace internal
+
+OVERRIDE_FEATURE_DEFAULT_STATES({{
+    {kSidePanelJourneys, base::FEATURE_DISABLED_BY_DEFAULT},
+}});
```

Comment:
can we please keep one `OVERRIDE_FEATURE_DEFAULT_STATES` instance per `.cc` file? they are cheap, but not free.

Reply:
Done in 58964ea4902eaac0cdeadb76b36f1a222ad6fe9b